facia-json/src/main/scala/com/gu/facia/client/models/Config.scala

```diff
@@ -1,6 +1,5 @@
 package com.gu.facia.client.models
 
-import com.sun.corba.se.spi.legacy.interceptor.UnknownType
```

Comment:
To double check – this is something we found was unused, so we're tidying up?

Reply:
I think this may have been added from a previous commit as there is no unknownType in this file.
I assume my editor may have deleted unused imports?

It is running locally without errors in the log.